### PR TITLE
Adding Continuous build badges.

### PR DESCRIPTION
**Pull request by av8ramit**
Adding continuous build badges to TensorFlow Addons.

### CODE REVIEW

**Comment by av8ramit** [comment]
@karmel any thoughts?

**Comment by seanpmorgan** [comment]
@av8ramit This is great thanks! Just a quick question:
1) Is there anyway we can add alerts (e.g. email) for failing builds?

File: README.md [hunk 1]
```diff
@@ -52,6 +52,17 @@ maintainable, SIG-Addons will perform periodic reviews and deprecate
 contributions which will be slated for removal. More information will
 be available after we submit a formal request for comment.
 
+## Continuous build status
+
+### Official Builds
+
+| Build Type      | Status |
```
Comment (seanpmorgan) [inline README.md]: IMO this can probably get moved up to right below the other badges

Comment (av8ramit) [reply README.md]: Done.

**Comment by av8ramit** [comment]
Yes, there is a way to modify the email policy. I'll let @karmel decide how best to proceed with that.

**Comment by facaiy** [comment]
> Is there anyway we can add alerts (e.g. email) for failing builds?

+ 1 . But which mailing list to receive the failing alert mail? Would it be too noisy for addons-dev?

File: README.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 [![PyPI Status Badge](https://badge.fury.io/py/tensorflow-addons.svg)](https://pypi.org/project/tensorflow-addons/)
 [![Gitter chat](https://img.shields.io/badge/chat-on%20gitter-46bc99.svg)](https://gitter.im/tensorflow/sig-addons)
 
+## Continuous build status
```
Comment (facaiy) [inline README.md]: Looks great! How about removing the line to leave room for introduction below?

**Comment by karmel** [comment]
@ewilderj -- can we add an addons alias for failure emails? We can send to an alias like addons-dev+testing@, but that puts the onus on developers to filter their emails accordingly, and I suspect that will scare some potential community members away.

@av8ramit -- Thanks! This LGTM; I will approve here, as the email question would go in a separate place anyhow.

**Comment by ewilderj** [comment]
Yes, I think I can setup addons-testing@ -- that work for you, @seanpmorgan ?

File: README.md (hunk 2 above)
Comment (av8ramit) [reply README.md]: Hey @facaiy I'm a bit confused by what you mean? Should I just move the introduction to the top after the Gitter chat?

**Comment by seanpmorgan** [comment]
@ewilderj that'd be great... thanks!
@facaiy I removed the line and I think it looks good... good to merge?

**Comment by ewilderj** [comment]
here we go -- https://groups.google.com/a/tensorflow.org/forum/#!forum/addons-testing

**Pull request merged**